### ISSUE TITLE
Support for reading from external links

### ISSUE THREAD

**Issue by GFleishman** [fiji/HDF5_Vibez#6]
As far as I can tell, HDF5_Vibez does not support reading from [external links](https://support.hdfgroup.org/HDF5/doc/RM/RM_H5L.html#Link-CreateExternal). I would like to add this functionality. I've forked the repo, will do my best, and submit a pull request when/if I come up with something that works. Any pointers from existing contributors on this matter would be a big help (e.g. which file, which section of which file, would be most appropriate to begin looking at). Some documentation of the issue:

Suppose I have a .h5 file which contains only external links; when I attempt to open such a file in Fiji with File --> import --> hdf5, I see the following:

<img width="894" alt="screen shot 2018-04-11 at 3 05 57 pm" src="https://user-images.githubusercontent.com/8507206/38637684-e6b197dc-3d99-11e8-8a62-001f11996d6e.png">

So, HDF5_Vibez knows the external links are there, but does not make them available for reading.
On the other hand, HDF5 compass can traverse the links in the same file no problem:

<img width="824" alt="screen shot 2018-04-11 at 3 07 28 pm" src="https://user-images.githubusercontent.com/8507206/38637768-246138d0-3d9a-11e8-93c3-cea494cccdaf.png">

So, I think the file is ok. FYI, I created the file using h5py:

`#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import os
import h5py

# get all paths, working directory and hdf5 file names
cwd = os.getcwd()
h5_files = os.listdir(cwd)
h5_files.sort()
h5_files = [s for s in h5_files if s.endswith('.h5')]

# create container object to write external links to
container_name = 'container.h5'
container = h5py.File(container_name, 'w')

# loop through h5py files and add them as external links
for i, h5_file in enumerate(h5_files):
    key = 'timepoint{}_extlink.h5'.format(i)
    container[key] = h5py.ExternalLink(cwd+'/'+h5_file, '/default')

`

Use case:
I have *thousands* of separate .h5 files, each a separate time point from the same time series experiment. I don't want to build a new .h5 file, rewriting the separate time points as groups, as this will take forever and the file will be huge. So, creating container files with external links to just the subset of time points that I want, and then using that to access those time points, seems like a crafty solution... but I need to be able to load and view the results.

**Comment by GFleishman** [comment]
Solved it! Was actually simpler than I thought to add this functionality. Added an EXTERNAL_LINK case to the switch statement in HDF5_Reader_Vibez.java recursiveGetInfo() function. I tested it against a container file with external links to datasets in other .h5 files, it works. I also tested loading .h5 files without external links, which still works fine. I have not tested every possible external link type (e.g. external links to groups) but this could certainly be done if required for merging.

I've submitted my solution as a pull request.
